buildFonts.sh

```diff
@@ -119,10 +74,7 @@ if [[ $(docker images "$IMAGE" | wc -l) -lt 2 ]]; then
 fi
 
 CMDS="set -ex
-test -f MathJax-dev.tar.gz || wget -O MathJax-dev.tar.gz '${URL}'
-mk=\$(tar tf MathJax-dev.tar.gz | grep 'fonts/OTF/TeX/Makefile\$')
 tar xf MathJax-dev.tar.gz
```

Comment:
Unpacking the `tar` file is still necessary.  In the future we might want to look at mounting the `src/` folder somewhere inside the docker.  I've had troubles mounting folders on OS X in the past that's why I'm avoiding it at the moment.

Reply:
Ah, I see, that's within the Docker.  Makes sense.